checks/network_checks.py

```diff
@@ -117,7 +128,7 @@ def restart_pool(self):
     def check(self, instance):
         if not self.pool_started:
             self.start_pool()
-        if threading.activeCount() > 5 * self.pool_size + 5: # On Windows the agent runs on multiple threads so we need to have an offset of 5 in case the pool_size is 1
+        if threading.activeCount() > 5 * _global_current_pool_size + 1: 
```

Comment:
It doesn't, in python the need for the global keyword is not needed when you read the value of a variable.